owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/PassCodeManager.kt

```diff
@@ -0,0 +1,111 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Juan Carlos Garrote Gascón
+ *
+ * Copyright (C) 2021 ownCloud GmbH.
+ * <p>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ * <p>
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * <p>
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.presentation.ui.security
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import android.os.PowerManager
+import android.os.SystemClock
+import android.preference.PreferenceManager
+import com.owncloud.android.MainApp.Companion.appContext
+import com.owncloud.android.authentication.BiometricManager
+
+object PassCodeManager {
+
+    var exemptOfPasscodeActivites: MutableSet<Class<*>> = mutableSetOf(PassCodeActivity::class.java)
+        private set
+
+    private var timestamp = 0L
+    private var visibleActivitiesCounter = 0
+
+    // keeping a "low" positive value is the easiest way to prevent the pass code is requested on rotations
+    private val PASS_CODE_TIMEOUT = 1000
```

Comment:
This one could be marked as a constant, couldn't it?
```suggestion
    private const val PASS_CODE_TIMEOUT = 1_000
```